jacodb-ets/src/main/kotlin/org/jacodb/ets/dto/Convert.kt

```diff
@@ -430,6 +430,14 @@ fun convertToEtsClass(classDto: ClassDto): EtsClass {
         file = null, // TODO
     )
 
+    val superclassSignature = classDto.superClassName?.takeIf { it != "" }?.let { spName ->
```

Comment:
Why `spName`? Maybe just `name`?

Reply:
Not to overlap with potential "name" variable in a higher scope